.github/workflows/ci.yml

```diff
@@ -84,7 +84,7 @@ jobs:
             jinahub/hubble-normalizer:v0.1.1 . --jina-version=2
             sudo chown -R $(id -u ${USER}):$(id -g ${USER}) . ; fi
       - name: Run unit tests
-        run: pytest -s -v -m "not gpu"
+        run: python -m pytest -s -v -m "not gpu"
```

Comment:
What is the motivation for this?

Reply:
It's actually the proper way of adding the executor dir path to `sys.path` during testing (so that the executor will be imported). 

Currently what we have works only because we have `test/__init__.py` file, and this for some reason causes pytest to add the executor directory. In the future, pytest will not do that anymore (see `importlib` [here](https://docs.pytest.org/en/6.2.x/pythonpath.html#import-modes)).

In any case, this change has no effect on how things work now - as pytest [documentation](https://docs.pytest.org/en/6.2.x/usage.html#cmdline) explains, adding current dir to `sys.path` is pretty much the only difference.